src/explore-education-statistics-frontend/src/components/CookieBanner.tsx

```diff
@@ -12,38 +13,95 @@ interface Props {
+  }, [isBannerSeen]);
 
-  return isMounted && getCookie('bannerSeen') !== 'true' ? (
+  return isMounted && isVisible ? (
     <div className={styles.container}>
       <div
-        className={classNames('govuk-width-container', 'dfe-width-container', {
+        className={classNames('govuk-cookie-banner', 'govuk-width-container', {
           'dfe-width-container--wide': wide,
         })}
+        role="region"
+        aria-label="Cookies on Explore Education Statistics"
       >
-        <p>
-          <span>GOV.UK uses cookies to make the site simpler.</span>{' '}
-          <ButtonText
-            onClick={() => {
-              setBannerSeenCookie(true);
-              if (typeof getCookie('disableGA') === 'undefined') {
-                setGADisabledCookie(false);
-              }
-            }}
+        {typeof getCookie('disableGA') === 'undefined' ? (
+          <div className="govuk-cookie-banner__message govuk-width-container">
+            <div className="govuk-grid-row">
+              <div className="govuk-grid-column-two-thirds">
+                <h2 className="govuk-cookie-banner__heading govuk-heading-m">
+                  Cookies on Explore Education Statistics
+                </h2>
+
+                <div className="govuk-cookie-banner__content">
+                  <p>
+                    We use some essential cookies to make this service work.
+                  </p>
+                  <p>
+                    We’d also like to use analytics cookies so we can understand
+                    how you use the service and make improvements.
+                  </p>
+                </div>
+              </div>
+            </div>
+
+            <ButtonGroup>
+              <Button
+                onClick={() => {
+                  setBannerSeenCookie(true);
+                  setGADisabledCookie(false);
+                }}
+              >
+                Accept analytics cookies
+              </Button>{' '}
+              <Button
+                onClick={() => {
+                  setBannerSeenCookie(true);
+                  setGADisabledCookie(true);
+                }}
+              >
+                Reject analytics cookies
+              </Button>{' '}
+              <Link to="/cookies">View cookies</Link>
+            </ButtonGroup>
+          </div>
+        ) : (
+          <div
+            className="govuk-cookie-banner__message govuk-width-container"
+            role="alert"
           >
-            Accept Cookies
-          </ButtonText>{' '}
-          or{' '}
-          <Link to="/cookies">
-            find out more about cookies and cookie settings
-          </Link>
-          .
-        </p>
+            <div className="govuk-grid-row">
+              <div className="govuk-grid-column-two-thirds">
+                <div className="govuk-cookie-banner__content">
+                  <p>
+                    You’ve{' '}
+                    {getCookie('disableGA') === 'false'
```

Comment:
One last thing - would be good to make this a single template string to avoid it being spread across multiple lines. This makes it easier to write a UI test for this in the future.